csharp-design-patterns/UnitOfWork/UnitOfWork/DataAccess/Dapper/DapperContext.cs

```diff
@@ -0,0 +1,27 @@
+using System.Data.SqlClient;
+using Dapper;
+using UnitOfWork.Entities;
+
+namespace UnitOfWork.DataAccess.Dapper;
+
+public sealed class DapperContext : IDatabase
+{
+    private readonly SqlConnection _sqlConnection;
+
+    public DapperContext(SqlConnection sqlConnection)
+    {
+        _sqlConnection = sqlConnection;
+    }
+
+    public async Task<ITransaction> BeginTransactionAsync()
+    {
+        var sqlTransaction = await _sqlConnection.BeginTransactionAsync();
+        return new DapperTransaction(sqlTransaction);
```

Comment:
Add a blank line between the code and the ``return` statement

Reply:
done